source/common/common/logger.h

```diff
@@ -343,6 +343,11 @@ class Registry {
    */
   static void setLogFormat(const std::string& log_format);
 
+  /**
+   * Sets the log format for a specific logger.
+   */
+  static void setLogFormatForLogger(spdlog::logger* const logger, const std::string& log_format);
```

Comment:
can we either update these to be `setLogFormatForComponentLoggers`, same for setLogLevel or move `setLogFormatForLogger` elsewhere since that object focuses more on Component level logging, preferring the latter

Reply:
This is like a util function, move it out.